content/es/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- raelga
+title: Componentes de Kubernetes
+content_type: concept
+weight: 20
+card:
+  name: concepts
+  weight: 20
+---
+
+<!-- overview -->
+
+Este documento describe los distintos componentes que
+son necesarios para operar un clúster de Kubernetes.
+
+<!-- body -->
+
+## Componentes maestros
```

Comment:
```suggestion
## Plano de control
```

Reply:
`master` wording removed in https://github.com/kubernetes/website/pull/22259/commits/40ca6e686e6631cd400b0db11701afa8e3f31091